use-cases/credit_risk/risk_bucketing.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>The other mentions use "Gap analysis" - <code>GAP</code> -&gt; <code>Gap</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated GAP to Gap in all cases for consistency